### PR TITLE
Bring up tvOS x64 staging pipeline leg

### PR DESCRIPTION
**Pull request by steveisok**
Contributes to https://github.com/dotnet/runtime/issues/33381

### CODE REVIEW

File: src/libraries/sendtohelixhelp.proj [hunk 1]
```diff
@@ -210,15 +210,25 @@
     <Message Importance="High" Text="Using WorkItemArchiveWildCard: $(WorkItemArchiveWildCard)" />
     <Message Importance="High" Text="Using Timeout: $(_workItemTimeout)" />
 
+    <PropertyGroup Condition="'$(TargetOS)' == 'iOS' or '$(TargetOS)' == 'tvOS' or '$(TargetOS)' == 'iOSSimulator' or '$(TargetOS)' == 'tvOSSimulator'">
+      <AppleTestTarget Condition="'$(TargetOS)' == 'iOSSimulator'">ios-simulator-64</AppleTestTarget>
+      <AppleTestTarget Condition="'$(TargetOS)' == 'tvOSSimulator'">tvos-simulator</AppleTestTarget>
+      <AppleTestTarget Condition="'$(TargetOS)' == 'iOS'">ios-device</AppleTestTarget>
+      <AppleTestTarget Condition="'$(TargetOS)' == 'tvOS'">tvos-simulator</AppleTestTarget>
```
Comment (premun) [inline src/libraries/sendtohelixhelp.proj:217]:
```suggestion
      <AppleTestTarget Condition="'$(TargetOS)' == 'tvOS'">tvos-device</AppleTestTarget>
```
?

**Review by akoeplinger — APPROVED** [review]

File: eng/pipelines/libraries/helix-queues-setup.yml [hunk 2]
```diff
@@ -105,10 +105,14 @@ jobs:
...
 
-    # iOS x64/x86
-    - ${{ if in(parameters.platform, 'iOSSimulator_x64', 'iOSSimulator_x86') }}:
+    # iOS simulator x64/x86
+    - ${{ if in(parameters.platform, 'iOSSimulator_x64', 'iOSSimulator_x86', 'tvOSSimulator_x64') }}:
       - OSX.1015.Amd64.Open
 
+    # iOS devices
+    - ${{ if in(parameters.platform, 'iOS_arm64', 'tvOS_arm64') }}:
+      - OSX.1015.Amd64.Iphone.Open
```
Comment (premun) [inline eng/pipelines/libraries/helix-queues-setup.yml:114]: `tvOS_arm64` should probably go to `OSX.1015.Amd64.AppleTV.Open`

Comment (akoeplinger) [reply eng/pipelines/libraries/helix-queues-setup.yml:114]: Yup.

**Review by premun — APPROVED** [review]

File: eng/pipelines/libraries/helix-queues-setup.yml (hunk 2 above)
Comment (premun) [reply eng/pipelines/libraries/helix-queues-setup.yml:114]: Could also be turned to `eq` instead of `in`

**Comment by steveisok** [comment]
Failures are unrelated https://github.com/dotnet/runtime/issues/50520

**Pull request merged**